### PR TITLE
Adding Bahase Malaysia

### PR DESCRIPTION
**Pull request by Ondoz**

### CODE REVIEW

File: src/Languages/lang.my.php [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+    'language_name' => 'Melayu',
+    'language'      => 'Gilang',
+    'per_page'      => 'Item per Halaman',
+
+    // Log in:
+    'log_in_to_app' => 'Masukkan Sensor PHP',
+    'login_error' => 'E-mel atau Kata Laluan yang Salah',
+    'forgotten_password_link' => 'Lupa Kata Laluan?',
+    'reset_emailed' => 'Kami telah menghantar pautan set semula kata laluan ke e-mel anda.',
+    'reset_header' => '<strong> Jangan risau! </strong> <br> Masukkan alamat e-mel anda dalam borang di bawah dan kami akan menghantar pautan tetapan semula kata laluan dalam e-mel anda.',
```
Comment (corpsee) [inline src/Languages/lang.my.php:13]: PHPCS: Line exceeds 120 characters; contains 188 characters

File: src/Languages/lang.my.php [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+    'error' => 'Error',
+    'skipped' => 'Dilangkau',
+    'trace' => 'Stack trace',
+    'manual_build' => 'Build Manual',
+
+    // Add/Edit Project:
+    'new_project' => 'Projek Baru',
+    'project_x_not_found' => 'Projek dengan ID% d tidak dijumpai.',
+    'project_details' => 'Butiran Projek',
+    'public_key_help' => 'Untuk lebih mudah untuk memulakan, kami telah menghasilkan pasangan kunci SSH untuk anda gunakan atau projek ini. Untuk menggunakannya, cukup tambahkan kunci awam berikut ke bahagian "kunci implement" pada platform hosting kod sumber pilihan anda. ',
```
Comment (corpsee) [inline src/Languages/lang.my.php:98]: PHPCS: Line exceeds 120 characters; contains 276 characters

File: src/Languages/lang.my.php [hunk 3]
```diff
@@ -0,0 +1,445 @@
...
+
+    'repo_name' => 'Nama / URL repositori (Remote) atau Laluan (Local)',
+    'project_title' => 'Nama Projek',
+    'project_private_key' => 'Kunci Peribadi yang digunakan untuk mengakses repositori
+                                (kosongkan untuk kawasan tempatan dan / atau tanpa nama) ',
+    'build_config' => 'Konfigurasikan binaan Sensor PHP untuk projek ini
+                                (jika anda tidak dapat menambahkan fail .php-censor.yml di repositori projek) ',
+    'default_branch' => 'Nama cawangan lalai',
+    'default_branch_only' => 'Bina cawangan lalai sahaja',
+    'overwrite_build_config' => 'Ganti fail konfigurasi di repositori dengan konfigurasi dalam pangkalan data? Sekiranya kotak centang tidak dicentang, konfigurasi dalam pangkalan data akan digabungkan dengan fail konfigurasi. ',
```
Comment (corpsee) [inline src/Languages/lang.my.php:119]: PHPCS: Line exceeds 120 characters; contains 229 characters

File: src/Languages/lang.my.php [hunk 4]
```diff
@@ -0,0 +1,445 @@
...
+    'source_manual_rebuild_console'        => 'Rebuild for %s (from CLI)',
+    'source_periodical'                    => 'Periodical',
+    'source_webhook_push'                  => 'Webhook (Push)',
+    'source_webhook_pull_request_created'  => 'Webhook (Created pull request)',
+    'source_webhook_pull_request_updated'  => 'Webhook (Updated pull request)',
+    'source_webhook_pull_request_approved' => 'Webhook (Approved pull request)',
+    'source_webhook_pull_request_merged'   => 'Webhook (Merged pull request)',
+
+    'webhooks' => 'Webhooks',
+    'webhooks_help_github' => 'Untuk membina projek ini secara automatik apabila komit baru didorong, tambahkan URL di bawah
```
Comment (corpsee) [inline src/Languages/lang.my.php:164]: PHPCS: Line exceeds 120 characters; contains 124 characters

File: src/Languages/lang.my.php [hunk 5]
```diff
@@ -0,0 +1,445 @@
...
+                                sebagai "Webhook" baru di <a href="https://github.com/%s/settings/hooks"> Webhooks
+                                dan Perkhidmatan </a> bahagian repositori GitHub anda. ',
+
+    'webhooks_help_gitlab' => 'Untuk membina projek ini secara automatik apabila komit baru didorong, tambahkan URL di bawah
```
Comment (corpsee) [inline src/Languages/lang.my.php:168]: PHPCS: Line exceeds 120 characters; contains 124 characters

File: src/Languages/lang.my.php [hunk 6]
```diff
@@ -0,0 +1,445 @@
...
+    sebagai "URL WebHook" di bahagian Hook Web di repositori GitLab anda. ',
+
+    'webhooks_help_gogs' => 'Untuk membina projek ini secara automatik apabila komit baru didorong, tambahkan URL di bawah
```
Comment (corpsee) [inline src/Languages/lang.my.php:171]: PHPCS: Line exceeds 120 characters; contains 122 characters

File: src/Languages/lang.my.php [hunk 7]
```diff
@@ -0,0 +1,445 @@
...
+                            sebagai "URL WebHook" di bahagian Web Hooks di repositori Gogs anda. ',
+
+    'webhooks_help_bitbucket' => 'Untuk membina projek ini secara automatik apabila komit baru didorong, tambahkan URL di bawah
```
Comment (corpsee) [inline src/Languages/lang.my.php:174]: PHPCS: Line exceeds 120 characters; contains 127 characters

File: src/Languages/lang.my.php [hunk 8]
```diff
@@ -0,0 +1,445 @@
...
+    'only_old' => 'Hanya ralat lama',
+    'new_errors' => 'Kesalahan baru',
+    'total_errors' => 'Kesalahan',
+
+    'committed_by_x' => 'Dilakukan oleh% s',
+    'commit_id_x' => 'Komit:% s',
+
+    'chart_display' => 'Carta ini akan dipaparkan setelah binaan selesai.',
+
+    'build'   => 'Build',
```
Comment (corpsee) [inline src/Languages/lang.my.php:211]: Copy and paste detected:

```
    'build'   => 'Build',
    'lines'   => 'Lines',
    'classes' => 'Classes',
    'methods' => 'Methods',
    'comment_lines' => 'Comment lines',
    'noncomment_lines' => 'Non-Comment lines',
    'logical_lines' => 'Logical Lines',
    'lines_of_code' => 'Lines of code',
    'coverage' => 'PHPUnit code coverage',
    'build_log' => 'Build log',
    'quality_trend' => 'Quality trend',
    'codeception_errors' => 'Codeception errors',
    'php_mess_detector_warnings' => 'PHPMD warnings',
    'php_code_sniffer_warnings' => 'PHPCS warnings',
    'php_code_sniffer_errors' => 'PHPCS errors',
    'phan_warnings' => 'Phan warnings',
    'php_cs_fixer_warnings' => 'PHP CS Fixer warnings',
    'php_parallel_lint_errors' => 'Lint errors',
    'php_unit_errors' => 'PHPUnit errors',
    'php_cpd_warnings' => 'PHP Copy/Paste Detector warnings',
    'php_docblock_checker_warnings' => 'Missing docblocks',
    'php_tal_lint_warnings' => 'PHP Tal Lint warnings',
    'php_tal_lint_errors' => 'PHP Tal Lint errors',
    'behat_warnings' => 'Behat warnings',
    'sensiolabs_insight_warnings' => 'Sensiolabs Insight warnings',
    'technical_debt_warnings' => 'Technical Debt warnings',
    'issues' => 'Issues',
    'merged_branches' => 'Merged branches',

    'codeception_feature' => 'Feature',
    'codeception_suite' => 'Suite',
    'codeception_time' => 'Time',
    'codeception_synopsis' => '<strong>%1$d</strong> tests carried out in <strong>%2$f</strong> seconds.
                               <strong>%3$d</strong> failures.',
    'suite' => 'Suite',
    'test'  => 'Test',
    'file' => 'File',
    'line' => 'Line',
    'class' => 'Class',
    'method' => 'Method',
    'message' => 'Message',
    'start' => 'Start',
    'end' => 'End',
    'from' => 'From',
    'to' => 'To',
    'result' => 'Result',
    'ok' => 'OK',
    'took_n_seconds' => 'Took %d seconds',
    'build_started' => 'Build Started',
    'build_finished' => 'Build Finished',
    'test_message' => 'Message',
    'test_no_message' => 'No message',
    'test_success' => 'Successful: %d',
    'test_fail' => 'Failures: %d',
    'test_skipped' => 'Skipped: %d',
    'test_error' => 'Errors: %d',
    'test_todo' => 'Todos: %d',
    'test_total' => '%d test(s)',

    // Users
    'name' => 'Name',
    'password_change' => 'Password (leave blank if you don\'t want to change)',
    'save' => 'Save &raquo;',
    'update_your_details' => 'Update your details',
    'your_details_updated' => 'Your details have been updated.',
    'add_user' => 'Add User',
    'is_admin' => 'Is Admin?',
    'yes' => 'Yes',
    'no' => 'No',
    'edit' => 'Edit',
    'edit_user' => 'Edit User',
    'delete_user' => 'Delete User',
    'user_n_not_found' => 'User with ID %d does not exist.',
    'is_user_admin' => 'Is this user an administrator?',
    'save_user' => 'Save User',

    // Settings:
    'settings_saved' => 'Your settings have been saved.',
    'settings_check_perms' => 'Your settings could not be saved, check the permissions of your config.yml file.',
    'settings_cannot_write' => 'PHP Censor cannot write to your config.yml file, settings may not be saved properly
                                until this is rectified.',
    'settings_github_linked' => 'Your GitHub account has been linked.',
    'settings_github_not_linked' => 'Your GitHub account could not be linked.',
    'build_settings' => 'Build Settings',
    'github_application' => 'GitHub Application',
    'github_sign_in' => 'Before you can start using GitHub, you need to <a href="%s">sign in</a> and grant
                            PHP Censor access to your account.',
    'github_app_linked' => 'PHP Censor is successfully linked to GitHub account.',
    'github_where_to_find' => 'Where to find these...',
    'github_where_help' => 'If you own the application you would like to use, you can find this information within your
                            <a href="https://github.com/settings/applications">applications</a> settings area.',

    'email_settings' => 'Email Settings',
    'email_settings_help' => 'Before PHP Censor can send build status emails,
                                you need to configure your SMTP settings below.',

    'application_id' => 'Application ID',
    'application_secret' => 'Application Secret',

    'smtp_server' => 'SMTP Server',
    'smtp_port' => 'SMTP Port',
    'smtp_username' => 'SMTP Username',
    'smtp_password' => 'SMTP Password',
    'from_email_address' => 'From Email Address',
    'default_notification_address' => 'Default Notification Email Address',
    'use_smtp_encryption' => 'Use SMTP Encryption?',
    'none' => 'None',
    'ssl' => 'SSL',
    'tls' => 'TLS',

    'failed_after' => 'Consider a build failed after',
    '5_mins' => '5 Minutes',
    '15_mins' => '15 Minutes',
    '30_mins' => '30 Minutes',
    '1_hour' => '1 Hour',
    '3_hours' => '3 Hours',

    // Plugins
    'cannot_update_composer' => 'PHP Censor cannot update composer.json for you as it is not writable.',
    'x_has_been_removed' => '%s has been removed.',
    'x_has_been_added' => '%s has been added to composer.json for you and will be installed next time
                            you run composer update.',
    'enabled_plugins' => 'Enabled Plugins',
    'provided_by_package' => 'Provided By Package',
    'installed_packages' => 'Installed Packages',
    'suggested_packages' => 'Suggested Packages',
    'title' => 'Title',
    'description' => 'Description',
    'version' => 'Version',
    'install' => 'Install &raquo;',
    'remove' => 'Remove &raquo;',
    'search_packagist_for_more' => 'Search Packagist for more packages',
    'search' => 'Search &raquo;',

    // Summary plugin
    'build-summary'  => 'Summary',
    'stage'          => 'Stage',
    'duration'       => 'Duration',
    'seconds'        => 'sec.',
    'plugin'         => 'Plugin',
    'stage_setup'    => 'Setup',
    'stage_test'     => 'Test',
    'stage_deploy'   => 'Deploy',
    'stage_complete' => 'Complete',
    'stage_success'  => 'Success',
    'stage_failure'  => 'Failure',
    'stage_broken'   => 'Broken',
    'stage_fixed'    => 'Fixed',
    'severity'       => 'Severity',

    'all_plugins'     => 'All plugins',
    'all_severities'  => 'All severities',
    'filters'         => 'Filters',
    'errors_selected' => 'Errors selected',

    'build_details'  => 'Build Details',
    'commit_details' => 'Commit Details',
    'committer'      => 'Committer',
    'commit_message' => 'Commit Message',
    'timing'         => 'Timing',
    'created'        => 'Created',
    'started'        => 'Started',
    'finished'       => 'Finished',

    // Update
    'update_app' => 'Update the database to reflect modified models.',
    'updating_app' => 'Updating PHP Censor database: ',
    'not_installed' => 'PHP Censor does not appear to be installed.',
    'install_instead' => 'Please install PHP Censor via php-censor:install instead.',

    // Create Build Command
    'add_to_queue_failed' => 'Build created successfully, but failed to add to build queue. This usually happens
                                when PHP Censor is set to use a beanstalkd server that does not exist,
                                or your beanstalkd server has stopped.',

    // Build Plugins:
    'passing_build' => 'Passing Build',
    'failing_build' => 'Failing Build',
    'log_output' => 'Log Output: ',


    // Error Levels:
    'critical' => 'Critical',
    'high' => 'High',
    'normal' => 'Normal',
    'low' => 'Low',

    // Plugins that generate errors:
    'php_mess_detector'    => 'PHP Mess Detector',
    'php_code_sniffer'     => 'PHP Code Sniffer',
    'php_unit'             => 'PHP Unit',
    'php_cpd'              => 'PHP Copy/Paste Detector',
    'php_docblock_checker' => 'PHP Docblock Checker',
    'composer'             => 'Composer',
    'php_loc'              => 'PHP LOC',
    'php_parallel_lint'    => 'PHP Parallel Lint',
    'email'                => 'Email',
    'atoum'                => 'Atoum',
    'behat'                => 'Behat',
    'campfire'             => 'Campfire',
    'clean_build'          => 'Clean Build',
    'codeception'          => 'Codeception',
    'copy_build'           => 'Copy Build',
    'deployer'             => 'Deployer',
    'env'                  => 'Env',
    'grunt'                => 'Grunt',
    'hipchat_notify'       => 'Hipchat',
    'irc'                  => 'IRC',
    'lint'                 => 'Lint',
    'mysql'                => 'MySQL',
    'package_build'        => 'Package Build',
    'pdepend'              => 'PDepend',
    'pgsql'                => 'PostgreSQL',
    'phan'                 => 'Phan',
    'phar'                 => 'Phar',
    'phing'                => 'Phing',
    'php_cs_fixer'         => 'PHP Coding Standards Fixer',
    'php_spec'             => 'PHP Spec',
    'shell'                => 'Shell',
    'slack_notify'         => 'Slack',
    'technical_debt'       => 'Technical Debt',
    'xmpp'                 => 'XMPP',
    'security_checker'     => 'SensioLabs Security Checker',

    'confirm_message' => 'Item will be permanently deleted. Are you sure?',
    'confirm_title'   => 'Item delete confirmation',
    'confirm_ok'      => 'Delete',
    'confirm_cancel'  => 'Cancel',
    'confirm_success' => 'Item successfully deleted.',
    'confirm_failed'  => 'Deletion failed! Server says: ',

    'public_status_title' => 'Public status',
    'public_status_image' => 'Status image',
    'public_status_page'  => 'Public status page',

```

**Comment by Ondoz** [comment]
@corpsee

**Comment by corpsee** [comment]
@Ondoz Thank you, Good job!

**Comment by Ondoz** [comment]
okay i will finish it

**Pull request merged**